packages/synapse-react-client/package.json

```diff
@@ -33,12 +33,12 @@
     "@brainhubeu/react-carousel": "1.19.26",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
-    "@mui/icons-material": "^5.14.16",
-    "@mui/material": "^5.14.17",
-    "@mui/system": "^5.14.17",
-    "@mui/utils": "^5.14.17",
-    "@mui/x-data-grid": "5.17.26",
-    "@mui/x-date-pickers": "^6.18.1",
+    "@mui/icons-material": "^5.15.10",
+    "@mui/material": "^5.15.10",
+    "@mui/system": "^5.15.9",
+    "@mui/utils": "^5.15.9",
+    "@mui/x-data-grid": "^6.19.4",
+    "@mui/x-date-pickers": "^6.19.4",
```

Comment:
Did you try out the updated date picker?

Reply:
No, but the upgrade here for `@mui/x-date-pickers` was a minor version bump that we could have picked up anyways with the current semver notation (we were already on v6). There should not be any breaking changes.